src/main/java/model/Race.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * 클래스 이름
+ * 레이스
+ * 버전 정보
+ * 1.0
+ * 날짜
+ * 2020.11.01
+ * 저작권 주의
+ */
+
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Race {
+    private List<Car> cars;
+    private int tryCount;
+
+    public void setCars (int carCount) {
+        cars = new ArrayList<>(carCount);
```

Comment:
```suggestion
        cars = new ArrayList<>();
```
carCount 를 생성자의 인자로 넘겨준 이유가 있을까요 ?